fuel/schemes.py

```diff
@@ -232,3 +232,17 @@ def get_request_iterator(self):
         indices = list(self.indices)
         self.rng.shuffle(indices)
         return iter_(indices)
+
+@add_metaclass(ABCMeta)
+class LimitedScheme(IterationScheme):
+    """Iterate over the sequence given an iterationScheme object
+        but in a limited number of excursions defined by times
+    """
+    def __init__(self, iteration_scheme, times, *args, **kwargs):
+        self.iteration_scheme = iteration_scheme
+        self.times = times
+        assert self.times >=1, "Error : you need a positiv number of mini batches"
+
+    def get_request_iterator(self):
+        it = self.iteration_scheme.get_request_iterator()
+        return iter_([it.next() for n in range(self.times)])
```

Comment:
Is there a reason why you're coercing `it.next() for n in range(self.times)` into a list?